main.py

```diff
@@ -1,10 +1,43 @@
+import json
+from django.utils.datetime_safe import datetime
 import init_django_orm  # noqa: F401
-
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for nickname, player_data in players_data.items():
+        race, _ = Race.objects.get_or_create(
+            name=player_data["race"]["name"],
+            defaults={"description": player_data["race"]["description"]}
+        )
+
+        guild = None
+        if player_data.get("guild"):
+            guild, _ = Guild.objects.get_or_create(
+                name=player_data["guild"]["name"],
+                defaults={"description": player_data["guild"]["description"]}
+            )
+
+        player, _ = Player.objects.get_or_create(
+            nickname=nickname,
+            defaults={
+                "email": player_data["email"],
+                "bio": player_data["bio"],
+                "race": race,
+                "guild": guild,
+                "created_at": datetime.now(),
```

Comment:
Consider using 'django.utils.timezone.now()' instead of 'datetime.now()' to ensure the 'created_at' field is timezone-aware, which is a best practice in Django applications.